WordPress/WordPressTest/Dashboard/DashboardBlazeCardCellViewModelTest.swift

```diff
@@ -3,27 +3,43 @@ import XCTest
 @testable import WordPress
 
 final class DashboardBlazeCardCellViewModelTest: CoreDataTestCase {
-    private let service = MockBlazeService()
-    private let store = MockDashboardBlazeStore()
+    private var service: MockBlazeService!
+    private var store: MockDashboardBlazeStore!
```

Comment:
I've also notified a [report](https://buildkite.com/organizations/automattic/analytics/suites/wordpress-ios/tests/0188feab-9e12-771f-b16b-3098dac6d63d?branch=trunk#failed) about `testCampaignRefresh()` being flaky, but I fail to see why. 

Yes, the setup is probably a bit more complicated than it needs to be. However, moving the initialization of the dependencies to `setUp()` should not affect the test, as the new instance of `XCTestCase` is created for every test. I usually try to align with the minimum required Xcode version by the App Store, which is currently Xcode 14.1. Do you know if there is a way to keep  Besides that, this test looks as simple as it gets.